thirdparty/CMakeMariaDBLists.txt

```diff
@@ -17,10 +17,10 @@ if(WIN32 AND NOT MARIADB_BUILD_SOURCE)
 			EXPECTED_HASH MD5=09d418c290109068a5bea136dafca36b)
 	endif()
 
-	if(NOT EXISTS "${MARIADB_MSI_DIR}/mariadb-connector-cpp-1.0.1-win64.msi" )
+	if(NOT EXISTS "${MARIADB_MSI_DIR}/mariadb-connector-cpp-1.0.2-win64.msi" )
 		message("Downloading mariadb connector/c++")
-		file(DOWNLOAD https://dlm.mariadb.com/1683453/connectors/cpp/connector-cpp-1.0.1/mariadb-connector-cpp-1.0.1-win64.msi
-			"${MARIADB_MSI_DIR}/mariadb-connector-cpp-1.0.1-win64.msi"
+		file(DOWNLOAD https://dlm.mariadb.com/2531525/Connectors/cpp/connector-cpp-1.0.2/mariadb-connector-cpp-1.0.2-win64.msi
+			"${MARIADB_MSI_DIR}/mariadb-connector-cpp-1.0.2-win64.msi"
 			EXPECTED_HASH MD5=548e743fbf067d21d42b81d958bf4ed7)
```

Comment:
Changing the version without changing the hash seems to have broken CI?

Reply:
Good catch but that's not what's up.  It's an issue with mariadb error ing out on the download.  I need to change the download link back but we won't have windows ci until they fix that.

Reply:
For reference, the hash that is returned here is just a null character hashed

Reply:
Changes were reverted.  Still worth noting that all CI is broken for windows (and mac for whatever reason) until mariadb addresses the ssl issue (or we could put in a workaround)

Reply:
Huh, interesting. What kind of SSL problem do they have? I don't see anything obvious on their end with https://dlm.mariadb.com/2531525/Connectors/cpp/connector-cpp-1.0.2/mariadb-connector-cpp-1.0.2-win64.msi, is there a tracking issue / status page on this somewhere?

Reply:
I've dug around and don't see any issue for it in their Jira, but that was a few days ago, I'll check again and make a ticket in there if there's isn't one

Reply:
Also, to be clear, the issue is that GitHub actions filter/block http requests, so this is just an issue here. And yeah finding the SSL info for that page is weird cause browsers are starting to hide that info from users now. But from what I can tell, there's either just no cert, a bad dummy cert, or an expired cert

Reply:
Afaict from online tools like https://dnschecker.org/ssl-certificate-examination.php and https://redirectdetective.com, the dlm.mariadb.com link serves a 302 redirect to storage.googleapi.com and both have a valid certificate chain.

Is there some region based IP resolution or MITM going on that GH Actions are hitting? I had a case with Visual Studio in the past where there US servers worked but the european ones were broken.